packages/complex-rubric/controller/src/defaults.js

```diff
@@ -140,19 +176,28 @@ const rubriclessDefaultConfiguration = {
   rubriclessInstruction: {
     settings: true,
     label: 'Instruction',
+    inputConfiguration: {
+      audio: { disabled: false },
+      video: { disabled: false },
+      image: { disabled: false },
+    },
   },
   maxMaxPoints: 100,
 };
 
 export default {
   model: {
-    rubricOptions: ['simpleRubric', 'multiTraitRubric'],
     rubricType: 'simpleRubric',
-    rubrics: { simpleRubric: rubricDefaultModel, multiTraitRubric: multiTraitDefaultModel, rubricless: rubriclessDefaultModel },
+    rubrics: {
+      [RUBRIC_TYPES.SIMPLE_RUBRIC]: rubricDefaultModel,
+      [RUBRIC_TYPES.MULTI_TRAIT_RUBRIC]: multiTraitDefaultModel,
+      [RUBRIC_TYPES.RUBRICLESS]: rubriclessDefaultModel,
+    },
   },
   configuration: {
-    multiTraitRubric: multiTraitDefaultConfiguration,
-    simpleRubric: rubricDefaultConfiguration,
-    rubricless: rubriclessDefaultConfiguration,
+    rubricOptions: ['simpleRubric', 'multiTraitRubric', 'rubricless'],
```

Comment:
I think there's a mistake here, so it might be just the right time to fix this.

defaults from controller should only have `model`, not `configuration`, because the controller doesn't have access in real like to `configuration`.
In complex-rubric it looks like we're not using this, anyway.